worker.go

```diff
@@ -254,6 +254,10 @@ func (w *worker) getAndDeleteUniqueJob(job *Job) *Job {
 		return nil
 	}
 
+	// JSON processing in Go differs from what Redis does (the order of values will be altered)
+	// we will use old JSON (not changing it), because the new JSON will have its data order altered
+	// for now we will stay using the old JSON, as we yet to have the solution to uniform the JSON
```

Comment:
if it is only the order of the value, why it causes a problem? Can you provide some examples?

Reply:
when doing LREM key 1 rawJSON in function `removeJobFromInProgress` it won't remove anything because rawJSON (as string) will differ from the one stored in redis because the order is altered (string comparison will still render it different strings)

Reply:
ok